### ISSUE TITLE
Browser support broken

### ISSUE THREAD

**Issue by atombender** [gf3/moment-range#81]
You have started using `const` in the non-dist version (starting with 1.2, I believe), which doesn't work with older browsers (for example, IE 10.x and earlier). We use `moment-range` in apps we transpile with Browserify and Babel, and minify with Google Closure. Closure will throw this:

```
js/app.js:32773: ERROR - Parse error. Unsupported syntax: CONST
const moment = require('moment');
^
 
js/app.js:32775: ERROR - Parse error. Unsupported syntax: CONST
const INTERVALS = {
^
 
2 error(s), 0 warning(s)
```

**Comment by gf3** [comment]
@atombender you should definitely use the `dist/` version of moment-range on the web. it's the [default in bower](https://github.com/gf3/moment-range/blob/master/bower.json#L4). also babel should translate the `const` to a `var` if you really want to use the `lib/` version (it's what i'm using).

**Comment by atombender** [comment]
We're using NPM, and IMHO this ought to just work, as with any NPM module:

```javascript
import 'moment-range';
```

I don't use Bower and I have the impression that it's a technology that's on the wane now that Browserify and Webpack are so mature.

As for Babel, it will not transpile anything in `node_modules` by default. Transpiling imported modules is awkward at best, as Babel's include pattern setting isn't additive. For example, instead of doing this:

    babel app.js

You now have to do:

    babel --only app.js,app/**/*,lib/**/*,bin/**/*,node_modules/moment-range app.js

or whatever. There is nothing in between. Suffice to say, the latter solution is not convenient.

**Comment by atombender** [comment]
By the way, if you don't like using `dist/` for NPM, you could add a `browser` section to `package.json`. Something like this should work:

```
"browser": {
  "./index.js": "./dist/moment-range.js"
}
```

Personally I'd prefer a single, consistent solution, though.

**Comment by gf3** [comment]
should be fixed by #83, thanks for your input @atombender

**Comment by atombender** [comment]
Awesome, thanks.
